experiment/posttest.json

```diff
@@ -0,0 +1,52 @@
+[
+    {
+        "question": "1. The above figure is the gate level implementation of:",
+        "answers": {
+            "a": "SR-Latch",
+            "b": "D-Latch",
+            "c": "T-Latch",
+            "d": "None of the above"
+        },
+        "correctAnswer": "b"
+    },
+    {
+        "question": "2. What kind of flip flop is generally preffered for constructing counters?",
```

Comment:
Fixed the spelling. We had not written this file so we didn't look into it in great detail earlier assuming that it would be correct. We have corrected it now